Source/Resources/MongoDB/ConnectionDetails.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using MongoDB.Driver;
+
+namespace Dolittle.Runtime.Resources.MongoDB
+{
+    /// <summary>
+    /// Represents the connection details for a MongoDB resource.
+    /// </summary>
+    /// <param name="ConnectionString">The connection string.</param>
+    /// <param name="DatabaseName">The MongoDB resource database name.</param>
+    public record ConnectionDetails(MongoUrl ConnectionString, string DatabaseName);
```

Comment:
Doesn't the MongoUrl already contain the DatabaseName?

Reply:
It does. I think that record isn't used anymore?

Reply:
File wasn't used anymore, so I got rid of it.